src/core/firebase.app.module.ts

```diff
@@ -1,40 +0,0 @@
-import { InjectionToken, } from '@angular/core';
-import { FirebaseAppConfig } from './';
-import firebase from '@firebase/app';
-
-import { FirebaseApp as FBApp } from '@firebase/app-types';
-import { FirebaseAuth } from '@firebase/auth-types';
-import { FirebaseDatabase } from '@firebase/database-types';
-import { FirebaseMessaging } from '@firebase/messaging-types';
-import { FirebaseStorage } from '@firebase/storage-types';
-import { FirebaseFirestore } from '@firebase/firestore-types';
-
-export const FirebaseAppConfigToken = new InjectionToken<FirebaseAppConfig>('FirebaseAppConfigToken');
-
-export class FirebaseApp implements FBApp {
```

Comment:
I would still like for us to provide the `FirebaseApp` token for DI. It does conflict with the SDK but this would be a breaking change to remove it.